customer-wallet/src/routes.js

```diff
@@ -64,20 +65,28 @@ function checkMnemonicExists(proceed) {
   proceed()
 }
 
-function preloadUser(nextState, proceed, { getState, dispatch }) {
+async function preloadUser(nextState, proceed, { getState, dispatch }) {
   const { data } = getState().data.profile
   const [ , nextUserType, ...rest ] = nextState.location.pathname.split('/')
 
   if (!data) {
-    dispatch(getProfile(nextUserType))
-      .then(({ userType }) => {
-        const nextPath = [ nextUserType || userType, ...rest ].join('/')
-        replace(`/${nextPath}`)
-      })
-      .catch(error => {
-        console.log('error_____', error)
-        proceed()
-      })
+    try {
```

Comment:
@Venumteam - You OK with using async/await here?
Actually @fazo96 why the change to async await here?

Reply:
I changed it to async/await since it made the code a little more readable and I saw async/await being used in other parts of the code. Let me know if you prefer to revert to promises

Reply:
@fazo96 async/await is ok. I just wanted Max to confirm this, and he did, so we are good with this change.